lib/paramedic.js

```diff
@@ -97,9 +97,14 @@ class ParamedicRunner {
             .fin((result) => {
                 isTestPassed = result;
                 logger.normal('Completed tests at ' + (new Date()).toLocaleTimeString());
-                // if we do --justbuild  or run on sauce,
-                // we should NOT do actions below
-                if (this.config.getAction() !== 'build' && !this.config.shouldUseSauce()) {
+
+                // If we run --shouldUseSauce immedatly fetch and return Sauce details.
+                if (this.config.shouldUseSauce()) {
```

Comment:
I believe your implementation is better than #112 and also fixes #111. So probably those two can be closed once this is merged.

Reply:
👍